### PR TITLE
replace multiple else/ifs with switch case

### PR DESCRIPTION
**Pull request by faisalsayed10**
Just a small contribution where I tried to replace those else-ifs with switch case which looks better in my opinion.

### CODE REVIEW

File: kibbeh/src/modules/layouts/DesktopLayout.tsx [hunk 1]
```diff
@@ -22,37 +22,44 @@ export const DesktopLayout: React.FC<MainLayoutProps> = ({
...
+			middle = (
       <>
         {children}
         {floatingRoomInfo}
       </>
-    );
+    	);
+			break;
+		default:
+			break;
```
Comment (overlisted) [inline kibbeh/src/modules/layouts/DesktopLayout.tsx:62]: you don't need this
```suggestion
```

**Review by overlisted — CHANGES_REQUESTED** [review]

File: kibbeh/src/modules/layouts/DesktopLayout.tsx (hunk 1 above)
Comment (faisalsayed10) [reply kibbeh/src/modules/layouts/DesktopLayout.tsx]: the default thing?

**Review by dhenson02 — CHANGES_REQUESTED** [review]
project indents are spaces not tabs. please don't mix

whatever IDE you use, download the editorconfig plugin and enable it to make this easier for you

File: kibbeh/src/modules/layouts/DesktopLayout.tsx [hunk 2]
```diff
@@ -22,37 +22,41 @@ export const DesktopLayout: React.FC<MainLayoutProps> = ({
   const screenType = useScreenType();
 
   let middle = null;
-
-  if (screenType === "3-cols") {
-    middle = (
+  
+  switch(screenType) {
```
Comment (dhenson02) [inline kibbeh/src/modules/layouts/DesktopLayout.tsx]: consistency - statements in project are padding with space around parenthesis - `switch (` not `switch(`

**Review by dhenson02 — COMMENTED** [review]
nothing major just something to keep in mind

File: kibbeh/src/modules/layouts/DesktopLayout.tsx (hunk 2 above)
Comment (faisalsayed10) [reply kibbeh/src/modules/layouts/DesktopLayout.tsx]: alright so I replaced the indentation with spaces

**Pull request merged**